src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/CustomerSettings/Groups/index.html.twig

```diff
@@ -0,0 +1,49 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+
+{% extends '@PrestaShop/Admin/layout.html.twig' %}
+
+{% set layoutHeaderToolbarBtn = {
+  add: {
+    href: path('admin_customer_groups_create'),
+    desc: 'Add new group'|trans({}, 'Admin.Shopparameters.Feature'),
+    icon: 'add_circle_outline',
+  }
+}
+%}
+{% set enableSidebar = true %}
```

Comment:
I have a preference for injecting these variables from the controller like here for example:
https://github.com/PrestaShop/PrestaShop/blob/205e83cc77c9ac17d9a8b959481f93922d9294b2/src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/ProfileController.php#L72-L80

But seems like both techniques are used, so not blocking It's just my personal preference